sumo-api/sumo/metrics/evaluator.py

```diff
@@ -86,7 +86,140 @@ def default_settings():
+            'chair',
+            'chair_set',
+            'chandelier',
+            'chessboard',
+            'clock',
+            'cloth',
+            'coffee_kettle',
+            'coffee_machine',
+            'coffee_table',
+            'column',
+            'computer',
+            'containers',
+            'cup',
+            'curtain',
+            'cutting_board',
+            'desk',
+            'dining_table',
+            'dishwasher',
+            'door',
+            'double_bed',
+            'dresser',
+            'dressing_table',
+            'dryer',
+            'fence',
+            'fireplace',
+            'fish_tank',
+            'fishbowl',
+            'floor',
+            'floor_lamp',
+            'food_processor',
+            'food_tray',
+            'fruit_bowl',
+            'game_table',
+            'garage_door',
+            'glass',
+            'goal_post',
+            'grill',
+            'gym_equipment',
+            'hanger',
+            'hanging_kitchen_cabinet',
+            'heater',
+            'household_appliance',
+            'iron',
+            'ironing_board',
+            'jug',
+            'kettle',
+            'kitchen_cabinet',
+            'kitchen_set',
+            'knife_rack',
+            'laptop',
+            'loudspeaker',
+            'magazines',
+            'microwave',
+            'mirror',
+            'motorcycle',
+            'office_chair',
+            'ottoman',
+            'outdoor_lamp',
+            'outdoor_seating',
+            'pan',
+            'partition',
+            'pedestal_fan',
+            'person',
+            'pet',
+            'piano',
+            'picture_frame',
+            'pillow',
+            'place_setting',
+            'plant',
+            'plates',
+            'playstation',
+            'pool',
+            'range_hood',
+            'range_oven',
+            'refrigerator',
+            'roof',
+            'rug',
+            'shelving',
+            'shoes_cabinet',
+            'shower',
+            'single_bed',
+            'sink',
+            'soap_dispenser',
+            'sofa',
+            'stairs',
+            'stand',
+            'stationary_container',
+            'stereo_set',
+            'switch',
+            'table_and_chair',
+            'table_lamp',
+            'telephone',
+            'television',
+            'toilet',
+            'towel_hanger',
+            'towel_rack',
+            'toy',
+            'trash_can',
+            'trinket',
+            'tv_stand',
+            'umbrella',
+            'utensil_holder',
+            'vacuum_cleaner',
+            'vase',
+            'wall',
+            'wall_lamp',
+            'wardrobe_cabinet',
+            'washer',
+            'water_dispenser',
+            'whiteboard',
+            'window',
+            'workplace',
+            'xbox',
+        ]
         density = 150  # gives 1 point about every 7.5 cm
```

Comment:
We probably need higher density. 7.5cm seems too coarse to get a fine-grained mesh. I understand that it makes things faster, but even in voxelization, our previous works used 5cm which I see as coarse.

Reply:
The density will be finer when the shape is more complex because there is a minimum of one point per triangle.  For now, this is a good trade off between speed and accuracy.  We can do more testing to see what the system can handle.  The primary time-sink is the routine that finds closest points.